src/presentation/fashion/CartDetail/CartLikePriceDetail/Promotions/Promotions.vue

```diff
@@ -0,0 +1,10 @@
+<style src="./Promotions.scss" lang="scss"></style>
+<script src="./Promotions.js"></script>
+
+<template>
+<span class="promo-title">Available promotions</span>
```

Comment:
Consider add translations here and in "ADD" below. But for me it's not rly required.